packages/tools/kda-cli/src/pact/coin.ts

```diff
@@ -0,0 +1,54 @@
+import { Reducer } from './pact.js';
+
+export type KAccount = `k:${string}` | string;
```

Comment:
Is this really useful? `| string` covers anything including `k:${string}`. Otherwise rename type `KAccount` to something like `AccountName`

Reply:
Not really useful, specially in this context. I added `| string` along the way as this tool needs to accept `w:` `r:` or any other type of accounts as well.

Reply:
Let's change it to `export type Account = string` then?